README.md

```diff
@@ -126,6 +126,29 @@ MicroShift keeps all its state in its data-dir, which defaults to `/var/lib/micr
 
 When starting the MicroShift for the first time the Kubeconfig file is created. If you need it for another user or to use externally the kubeadmin's kubeconfig is placed at `/var/lib/microshift/resources/kubeadmin/kubeconfig`.
 
+### Pulling Container Image From Private Registries
+
+MicroShift may not have the pull secret for the registry that you are trying to use. For example, MicroShift does not have the pull secret for registry.redhat.io. In order to use this registry, there are several approaches. The first approach is to use podman login,
+```sh
+podman login registry.redhat.io
+```
+
+Once the podman login is complete, MicroShift will be able to pull images from this registry. This approach works across name spaces.
+
+This approach assumes podman is installed. This might not be true for all MicroShift environments. For example, if MicroShift is installed through RPM, CRI-O will be installed as dependency, but no podman. In this case, one can choose to install podman separately, or use other approaches described below. 
+
+The second approach is to create a pull secret, then let the service account to use this pull secret. This approach works within a name space. For example, if the pull secret is stored in a json formatted file "secret.json",
+```sh
```

Comment:
edit `then link the secret to a service account in the namespace` (or, is it always `the default service account`? if so, `then link the secret to the default service account in the namespace`)